eng/Versions.props

```diff
@@ -231,7 +231,8 @@
     <SystemThreadingTasksExtensionsVersion>4.5.4</SystemThreadingTasksExtensionsVersion>
     <SQLitePCLRawbundle_greenVersion>2.0.4</SQLitePCLRawbundle_greenVersion>
     <UIAComWrapperVersion>1.1.0.14</UIAComWrapperVersion>
-    <MicrosoftVSSDKBuildToolsVersion>16.8.3036</MicrosoftVSSDKBuildToolsVersion>
+    <MicroBuildPluginsSwixBuildVersion>1.1.33</MicroBuildPluginsSwixBuildVersion>
+    <MicrosoftVSSDKBuildToolsVersion>17.0.39-dev17-g977fccd6</MicrosoftVSSDKBuildToolsVersion>
```

Comment:
Does this need to go into a vs-deps branch or does this continue to work in Dev16?

Reply:
This change is only going to dev17 branch.